clients/fides-js/src/lib/hooks/useMediaQuery.ts

```diff
@@ -14,9 +14,20 @@ export const useMediaQuery = (query: string) => {
     function handleChange(e: MediaQueryListEvent) {
       setMatches(e.matches);
     }
-    matchQueryList.addEventListener("change", handleChange);
+
+    if (matchQueryList.addEventListener) {
+      matchQueryList.addEventListener("change", handleChange);
+    } else {
+      // Supports Safari < 14
```

Comment:
I would suggest leaving this note out or updating it to be more generic. There's a good chance someone might see it and remove due to old browsers not being officially supported.
```suggestion
      // Older browser and test automation support
```